Parser/pegen/pegen.h

```diff
@@ -261,13 +266,17 @@ expr_ty _PyPegen_concatenate_strings(Parser *p, asdl_seq *);
 asdl_seq *_PyPegen_join_sequences(Parser *, asdl_seq *, asdl_seq *);
 int _PyPegen_check_barry_as_flufl(Parser *);
 mod_ty _PyPegen_make_module(Parser *, asdl_seq *);
+OperatorTermPair *_PyPegen_operator_term_pair(Parser *p, Token* oper, expr_ty factor);
+expr_ty _PyPegen_operator_precedence_expr(Parser *p, expr_ty factor, asdl_seq *terms);
 
 // Error reporting helpers
 expr_ty _PyPegen_get_invalid_target(expr_ty e);
 void *_PyPegen_arguments_parsing_error(Parser *, expr_ty);
 void *_PyPegen_nonparen_genexp_in_call(Parser *p, expr_ty args);
 
+// Automatically generated
```

Comment:
Hm, why not also generate get_binary_operator()?

Reply:
Because that is fixed IIUC, there is no relationship in the grammar between the AST items for binary operators (operator_ty) and the tokens so this needs to be hand-written in one place or another. As this is only used in `pegen.c` I opted to make it static